app/src/utils/validate.js

```diff
@@ -4,7 +4,8 @@ export function validateIpAddress(input) {
   return !!input.match(ipFormat);
 }
 
-export const QSLRegEx = /([a-zA-Z0-9]+)\[?(?:(@[()",@$=><!a-zA-Z0-9\-.|&:_]*|\**|\$\$[a-zA-Z0-9,=]+))\]?\{([*|[,@"=a-zA-Z0-9-]*)/;
+export const QSLRegEx = /([a-zA-Z0-9]+)\[?(?:(\@[\(\)\"\,\@\$\=\>\<\~\!a-zA-Z0-9\-\.\|\&\:_\^\*]*|\**|\$\$[a-zA-Z0-9\,\=]+))\]?\{([\*|[\,\@\"\=a-zA-Z0-9\-]*)/;
```

Comment:
the `\` is not needed in js.

Reply:
removed \ from regex string, please check